src/ensembl/src/content/app/entity-viewer/shared/helpers/entity-helpers.ts

```diff
@@ -191,3 +192,25 @@ export const getProteinXrefs = <
 
   return proteinXrefs;
 };
+
+export const getTranscriptMetadata = (
+  metadata: Pick<TranscriptMetadata, 'canonical' | 'mane'>
+) => {
+  const { canonical, mane } = metadata;
+  if (canonical && mane) {
+    return {
+      label: mane.label,
+      definition: mane.definition
+    };
+  } else if (canonical) {
+    return {
+      label: canonical.label,
+      definition: canonical.definition
+    };
+  } else if (mane) {
+    return {
+      label: mane.label,
+      definition: mane.definition
+    };
+  }
```

Comment:
Could be rewritten as:

```javascript
if (mane) {
    return {
      label: mane.label,
      definition: mane.definition
    };
  } else if (canonical) {
    return {
      label: canonical.label,
      definition: canonical.definition
    };
  } 
```

Reply:
Ah I copied this from `TranscriptQualityLabel.tsx` and didn't bother to check it properly. My bad.

I'll change this in that file too.

Reply:
✅